atmat/atphysics/Radiation/FDW.c

```diff
@@ -0,0 +1,572 @@
+/* FDW.c
+
+   mex-function to calculate integrated radiation diffusion matrix B defined in [2] 
+   for wiggler elements in MATLAB Accelerator Toolbox
+   O.Jimenez 3/08/18

```

Comment:
Has it really been initiated in 2018 ? You could mention your contribution…

Reply:
I will modify it and add my contribution to all files in the next commit.